src/screens/DiscoveryScreenV2/elements/Search.js

```diff
@@ -180,7 +181,7 @@ const DiscoverySearch = ({
             ref={discoverySearchBarRef}
             testID={TestIdConstant.discoveryScreenSearchBar}
             focusable={true}
-            autoFocus={true}
+            autoFocus={autoFocus}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Passing the `autoFocus` prop to the `TextInput` component allows control over whether the text input should automatically get focus or not. This is a good practice for enhancing usability.
```diff
- autoFocus={true}
+ autoFocus={autoFocus}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->